### DISCUSSION TITLE
1.6.1 or 1.7.0 predictions?

### DISCUSSION THREAD

**Discussion by Marc92020** [ScratchAddons/ScratchAddons#1094] [Discussion]
So we know what 1.6 will have right?
Lets see what we could possibly come up with for 1.6.1 or 1.7!

Predictions and replies:

deleted due to being "off topic"

**Comment by Creators-722** [comment]
please add new blocks in 1.6.1 and 1.7

**Reply by GrahamSH-LLK** [reply]
Rejected, non SA users couldn't use them.

**Reply by Marc92020** [reply]
but what if users without scratch addons just see the workarounds?

**Reply by GrahamSH-LLK** [reply]
That's literally impossible.

**Reply by Marc92020** [reply]
o but I saw that idea before.

**Comment by apple502j** [comment]
Predictions are offtopic.
